sound/soc/sof/topology.c

```diff
@@ -2229,7 +2227,7 @@ static int sof_process_load(struct snd_soc_component *scomp, int index,
 	}
 
 	process = (struct sof_ipc_comp_process *)
-		  sof_comp_alloc(swidget, &ipc_size, index, 0);
```

Comment:
this is actually a functional change, your commit message does not capture that. It's not just a replacement, it fixes an error that resulted in the process being scheduled on core0. You also need a Fixes tag for this, it'll make the upstream contribution easier.

Reply:
good point @plbossart let me change it.